integration/kubernetes/pod-liveness.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  labels:
+    test: liveness
+  name: liveness-exec
```

Comment:
please add the untrusted annotations:
```
annotations:
    io.kubernetes.cri-o.TrustedSandbox: "false"
    io.kubernetes.cri.untrusted-workload: "true"
```
e.g: https://github.com/kata-containers/tests/blob/master/integration/kubernetes/untrusted_workloads/busybox-template.yaml#L5-L7